compiler-core/src/build/package_compiler.rs

```diff
@@ -594,28 +603,28 @@ pub(crate) fn module_name(package_path: &Utf8Path, full_module_path: &Utf8Path)
 #[derive(Debug)]
 pub(crate) enum Input {
     New(UncompiledModule),
-    Cached(CachedModule),
+    Cached((CachedModule, CacheMetadata)),
```

Comment:
this doesn't need to be wrapped in a tuple, you're already in an enum tuple variant

```suggestion
    Cached(CachedModule, CacheMetadata),
```

Reply:
Thx for the suggestion! Fixed in commit #4789222ac8db5fe157576681547f5458f1508e76